### ISSUE TITLE
Taking screenshots of "Window without border" no longer works

### ISSUE THREAD

**Issue by kdvolder** [flameshot-org/flameshot#2647]
### Flameshot Version

Version info:
Flameshot v11.0.0 (11.0.0-2 Debian)
Compiled with Qt 5.15.2
linux: 5.15.0-35-generic
ubuntu: 22.04

### Installation Type

Linux, MacOS, or Windows Package manager (apt, pacman, eopkg, choco, brew, ...)

### Operating System type and version

Ubuntu 22.04

### Description

I'm trying to take a screenshot of just the 'contents' of a window. So I select the option to 'Grab the current Window' 
and make sure that 'Include pointer' and 'include the window border' are both *not* checked.

But despite the `include the window border` not being checked, I still end up with a screenshot that *does* include the window border. 

### Steps to reproduce

1. run 'flameshot gui'
2. Select 'Options...'
3. Select options as follows
  - Grab the whole screen (selected)
  - Grab after delay: 10 seconds
  - Include pointer (*not* selected)
  - Include the window border (*not* selected)
 4. Click 'Take Screenshot'
 5. Select a window
 6. Wait for 10 seconds or so

Observe that: a screenshot is taken that includes the window border.
Expected behavrio: a screenshot is taken that *does not* include the window border.

### System Information

1. Ubuntu 22.04
2. monitor config
![image](https://user-images.githubusercontent.com/1002156/172066051-1ed272a6-9427-44c2-8c8e-458c6765393b.png)
3. Linux
  - Desktop environment: Gnome
  - wayland or xorg?
 
```
$ loginctl show-session $(loginctl | grep $(whoami) | awk '{print $1}') -p Type
Type=wayland
```

**Comment by mmahmoudian** [comment]
Are you sure you are filing the bug report for the correct software?!?! We do not have any of the things implemented! We do **not** have:
- selecting a specific window
- inclusion/exclusion of border
- inclusion/exclusion of mouse pointer
- "Options..."

If you are 100% sure you are using Flameshot, please record a short video and show us exactly what you mean. We suggest using [peek](https://github.com/phw/peek).

**Comment by borgmanJeremy** [comment]
@mmahmoudian I suspect this is coming from the gnome-xdg-portal screen.

**Comment by kdvolder** [comment]
Recording: https://drive.google.com/file/d/13SlI2ypSmzmJTEobmkDVlKDMNAE_PmDs/view?usp=sharing

**Comment by kdvolder** [comment]
So when I do 'flaemshot gui' a window pops up on the screen with the described options. But the window that pops up isn't actually flameshot but some other tool?

**Comment by mmahmoudian** [comment]
@kdvolder Read this:

https://github.com/flameshot-org/flameshot/issues/2590#issuecomment-1122196178

**Comment by kdvolder** [comment]
Okay, I read some of that, thanks for the pointers. Yes, that really sucks, and I understand it is beyond your control. My apologies for wasting your time barking up the wrong tree.

**Comment by mmahmoudian** [comment]
@kdvolder No worries. Unfortunately it is out of our control. We tried to suggest an alternative approach to them, but as it is evident in those Gitlab threads, we hit a closed "door" every time we tried. Therefore, we have given up on the cause and have focused on things we can improve and fix. Feel free to open Feature requests, Bug Reports, and Pull Requests in future.

I personally do hope living in Gnome gets easier for you and other Gnome users.
